### DISCUSSION TITLE
code-conventies

### DISCUSSION THREAD

**Discussion by KoopReynders** [fdnd/docs.fdnd.nl#91] [Boksa]
Dit zijn de code-conventies die we de studenten willen leren. Er is vast een bron voor van generiek-naar-specifiek, 

Node & Express (server.js)
Volgorde van routes is hetzelfde als de sitemap/volgorde pagina;s
GET & POST routes van dezelfde route staan bij elkaar

HTML (EJS)
Volgorde van de HTML elementen is hetzelfde als de pagina structuur
Orden de HTML van groot-naar-klein, eerst de container elementen en daarna de geneste elementen

CSS
Volgorde van de CSS komt overeen met de volgorde van de HTML
Orden de CSS is van generiek-naar-specifiek, generieke styling staat bovenaan

**Comment by KoopReynders** [comment]
@joostf kan hier vast bij helpen?

**Comment by joostf** [comment]
Wij hanteren conventies bij FDND Agency, hier kan naar verwezen worden.

https://github.com/fdnd-agency/.github/wiki/Conventies
